Classes/CardIOViewController.mm

```diff
@@ -504,7 +504,7 @@ - (void)cardIOView:(CardIOView *)cardIOView didScanCard:(CardIOCreditCardInfo *)
     dataEntryViewController.manualEntry = self.context.suppressScannedCardImage;
 
     CGPoint newCenter = [self.view convertPoint:cardIOView.transitionView.cardView.center fromView:cardIOView.transitionView];
-    newCenter.y -= NavigationBarHeightForOrientation([[UIApplication sharedApplication] statusBarOrientation]);
+    newCenter.y -= NavigationBarHeightForOrientation(self.interfaceOrientation);
```

Comment:
Orientation stuff is tricky and often runs into issues with unexpected edge cases. I hope that you've tried every possible experiment!